tasks/presharedkey.yml

```diff
@@ -35,9 +35,17 @@
   delegate_to: localhost
 
 - name: Generate random {{ preshared_key_label }}
-  set_fact:
-    __ha_cluster_some_preshared_key: >
-      "{{ lookup('pipe', 'openssl rand -base64 {{ preshared_key_length }}') }}"
+  block:
+    - name: Generate {{ preshared_key_label }} using OpenSSL
+      command:
+        cmd: openssl rand -base64 {{ preshared_key_length | quote }}
+      register: __ha_cluster_openssl_rand
+      changed_when: no
```

Comment:
add `no_log: true` here

Reply:
`no_log: true` is set for the whole block containing this task. I went through all logs from CI and verified that output of this task is indeed hidden.